frappe/public/js/frappe/ui/notifications/notifications.js

```diff
@@ -204,34 +215,59 @@ frappe.ui.Notifications = class Notifications {
 	change_activity_status() {
 		if (this.$dropdown_list.find('.activity-status')) {
 			this.$dropdown_list.find('.activity-status').replaceWith(
-				`<a class="recent-item text-center text-muted" 
+				`<a class="recent-item text-center text-muted"
 					href="#List/Notification Log">
 					<div class="full-log-btn">${__('View Full Log')}</div>
 				</a>`
 			);
 		}
 	}
 
-	set_field_as_seen(docname, $el) {
+	set_field_as_read(docname, $el) {
 		frappe.call(
-			'frappe.desk.doctype.notification_log.notification_log.mark_as_seen',
+			'frappe.desk.doctype.notification_log.notification_log.mark_as_read',
 			{ docname: docname }
 		).then(()=> {
-			$el.removeClass('unseen');
+			$el.removeClass('unread');
 		});
 	}
 
-	explicitly_mark_as_seen(e, $target) {
+	explicitly_mark_as_read(e, $target) {
 		e.preventDefault();
 		e.stopImmediatePropagation();
-		let docname = $target.parents('.unseen').attr('data-name');
-		this.set_field_as_seen(docname, $target.parents('.unseen'));
+		let docname = $target.parents('.unread').attr('data-name');
+		this.set_field_as_read(docname, $target.parents('.unread'));
 	}
 
-	mark_as_seen(e, $target) {
+	mark_as_read(e, $target) {
 		let docname = $target.attr('data-name');
 		let df = this.dropdown_items.filter(f => docname.includes(f.name))[0];
-		this.set_field_as_seen(df.name, $target);
+		this.set_field_as_read(df.name, $target);
+	}
+
+	mark_all_as_read(e) {
+		e.stopImmediatePropagation();
+		this.$dropdown_list.find('.unread').removeClass('unread');
+		let unread_docnames = this.dropdown_items
```

Comment:
I think `Mark all as read` should mark all unread Notifications from the server side, not only the ones that are shown in the dropdown.